package.json

```diff
@@ -16,7 +16,7 @@
     "@fortawesome/fontawesome-free": "6.2.0",
     "@maplibre/maplibre-gl-leaflet": "0.0.17",
     "acorn": "8.8.1",
-    "adhocracy4": "liqd/adhocracy4#23e09419fe20c5a4f86ba0f9fa6ef33d268f5e92",
+    "adhocracy4": "liqd/adhocracy4#ec13ccd26008108a45379db95829655de241c5e5",
```

Comment:
mb is already on https://github.com/liqd/adhocracy4/commit/23e09419fe20c5a4f86ba0f9fa6ef33d268f5e92 so we probably don't need to update again ? :)

Reply:
oh yes, thanks, will remove the commit!